src/Main.java

```diff
@@ -0,0 +1,183 @@
+
+public class Main {
+
+
+    public static void main(String[] args) {
+
+        Farfetchd peter = new Farfetchd("Peter","","","",29,39);
+
+        peter.attack();
+        peter.makeNoise();
+
+        Unown womble = new Unown("Womble","","","",12,36,'G');
+
+        womble.healSelf();
+        womble.healSelf();
+        womble.attack();
+        womble.makeNoise();
+
+        Charjabug scrumple = new Charjabug("Scrumple","","","",8,15);
+
+        scrumple.attack();
+        scrumple.makeNoise();
+
+        Toxtricity rose = new Toxtricity("rose","","","",4,11,true);
+
+        rose.attack();
+        rose.makeNoise();
+    }
+
+}
+
+abstract class Pokémon{
+
+    private String name;
+    private String type1;
+    private String type2;
+    private String sound;
+    private int level;
+    private int hp;
+
+    public Pokémon(String name, String type1, String type2, String sound, int level, int hp) {
```

Comment:
In het geval dat je 2 types in je constructor hebt staan moet je ze elke keer meegeven, je had hier ook een tweede constructor kunnen aanmaken die String type2 niet vraagt.